updates/v2.0.2/extend_users_roles_table.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Winter\Test\Updates;
+
+use Schema;
+use Winter\Storm\Database\Updates\Migration;
+
+class ExtendUsersRolesTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('winter_test_users_roles', function($table)
+        {
+            $table->json('awards')->nullable();
```

Comment:
This one might fail depending on the DB used by the install,

Reply:
thanks, I updated my fork to use text() instead of json() for improved compatibiliy: https://github.com/hirisov/wn-test-plugin/tree/hirisov/pivot-repeater . The bug itself is present regardless of the column type.